### PR TITLE
feat(web-analytics): Add popover to learn more about channel types

### PR DESCRIPTION
**Pull request by robbie-c**
## Problem

Channel type is super valuable, but not very prominent and could do with some explanation of how it is calculated.

## Changes

Make channel the default option in the sources tile. Add a popover explaining what it is.

Video: https://github.com/PostHog/posthog/assets/2056078/f6bfd095-a16a-4245-b993-aac80fa314ec
Screenshot 2024-04-12 at 15 16 09

## Does this work well for both Cloud and self-hosted?

Yes

## How did you test this code?

Manually

### CODE REVIEW

File: frontend/src/scenes/web-analytics/WebDashboard.tsx [hunk 1]
```diff
@@ -249,6 +275,57 @@ export const WebTabs = ({
...
+                        />
+                    </div>
+                    <p className="text-sm text-gray-700">{description}</p>
+                    <div className="flex justify-end mt-4">
+                        <LemonButton
+                            to={docsURL}
+                            onClick={() => setIsOpen(false)}
+                            targetBlank={true}
+                            sideIcon={<IconOpenSidebar className="w-4 h-4" />}
+                            type="tertiary"
```
Comment (daibhin) [inline frontend/src/scenes/web-analytics/WebDashboard.tsx:310]: Same as below
```suggestion
```

File: frontend/src/scenes/web-analytics/WebDashboard.tsx [hunk 2]
```diff
@@ -249,6 +275,57 @@ export const WebTabs = ({
...
+                            icon={<IconX />}
+                        />
+                    </div>
+                    <p className="text-sm text-gray-700">{description}</p>
+                    <div className="flex justify-end mt-4">
+                        <LemonButton
+                            to={docsURL}
+                            onClick={() => setIsOpen(false)}
+                            targetBlank={true}
+                            sideIcon={<IconOpenSidebar className="w-4 h-4" />}
```
Comment (daibhin) [inline frontend/src/scenes/web-analytics/WebDashboard.tsx:309]: Out of curiosity did you intentionally add the size classes to this icon? Suggests that 1) icon sizes are incorrect on this size button or 2) this icon is the wrong default size

**Review by daibhin — APPROVED** [review]
Looks good. I couldn't find how to see it locally so a screenshot / gif could have helped a lot

File: frontend/src/scenes/web-analytics/WebDashboard.tsx [hunk 3]
```diff
@@ -116,6 +118,13 @@ const QueryTileItem = ({ tile }: { tile: QueryTile }): JSX.Element => {
             )}
         >
             {title && <h2 className="m-0 mb-3">{title}</h2>}
+            {docs && (
+                <LearnMorePopover
+                    docsURL="https://posthog.com/docs/data/channel-type"
+                    title="Channel Type"
+                    description="Channel type tells what kind of acquisition channel this traffic comes from, e.g. Paid Search or Organic Social"
+                />
```
[the remark this answers is not in the document]
Comment (robbie-c) [reply frontend/src/scenes/web-analytics/WebDashboard.tsx]: oops

File: frontend/src/scenes/web-analytics/WebDashboard.tsx (hunk 2 above)
Comment (robbie-c) [reply frontend/src/scenes/web-analytics/WebDashboard.tsx]: This was copied directly from ProductIntroduction.tsx but have removed it

**Pull request merged**